src/Converters/Converters/ValueConverterGroup.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Xamarin.Forms;
+
+namespace FormsCommunityToolkit.Converters
+{
+    /// <summary>
+    /// Conversion values using the sequential conversion of each of the converters from the first to the last.
+    /// </summary>
+    [ValueConversion(typeof(IEnumerator<IValueConverter>), typeof(IEnumerator<IValueConverter>))]
+    public class ValueConverterGroup : List<IValueConverter>, IValueConverter
+    {
+        public static ValueConverterGroup Instance { get; } = new ValueConverterGroup();
+
+        /// <summary>
+        /// Convert the value using a sequence of converters, from first to last.
+        /// </summary>
+        /// <param name="value"></param>
```

Comment:
Either provide useful explanation of each property, or you can leave these away. Right now it takes screen estate without added value.

Reply:
done